frontend/src/metabase/dashboard/analytics.ts

```diff
@@ -26,3 +26,10 @@ export const trackCardCreated = (type: CardTypes, dashboard_id: number) => {
     dashboard_id,
   });
 };
+
+export const trackCardMoved = (dashboardId: DashboardId) => {
+  trackSchemaEvent("dashboard", "1-1-2", {
```

Comment:
Here I used 1-1-2, should other events in this file be updated to that version too?

Reply:
Yes

Reply:
Thanks, I extracted it to a constant